src/software/ai/evaluation/ball_test.cpp

```diff
@@ -147,3 +147,49 @@ INSTANTIATE_TEST_CASE_P(Positions, BallPositionsInEnemyCornerOffField,
                         ::testing::Values(Point(4.6, 3), Point(4.6, -3), Point(5.0, 3),
                                           Point(5.0, -3), Point(4.5, -3.1),
                                           Point(4.0, -3.1)));
+TEST(DetectSuccessfulKickTest, ball_over_speed_threshold_and_no_direction_difference)
```

Comment:
```suggestion
TEST(HasBallBeenKickedTest, ball_over_speed_threshold_and_no_direction_difference)
```
^Rename all test names to match function name

Reply:
renamed test names  to match function name